src/sentry/api/endpoints/team_issue_breakdown.py

```diff
@@ -0,0 +1,54 @@
+import copy
+from datetime import timedelta
+
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import GroupHistory, GroupHistoryStatus, Project
+from sentry.models.grouphistory import ACTIONED_STATUSES
+
+
+class TeamIssueBreakdownEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Returns a dict of team projects, and a time-series dict of issue stat breakdowns for each.
+
+        Right now the stats we return are the count of reviewed issues and the total count of issues.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+        end = end.date() + timedelta(days=1)
+        start = start.date() + timedelta(days=1)
+        bucketed_issues = (
+            GroupHistory.objects.filter(
+                organization_id=team.organization_id,
+                status__in=[GroupHistoryStatus.UNRESOLVED] + ACTIONED_STATUSES,
+                project__in=project_list,
+                date_added__gte=start,
+                date_added__lte=end,
```

Comment:
You might need to add an additional index that includes date_added. Maybe `("date_added", "project", "status", "release")` or `("project", "date_added", "status", "release")`? Not sure on the ordering that will be most useful

Reply:
Will follow up with this